INSTALL.md

```diff
@@ -97,6 +97,11 @@ See [the rustc-dev-guide for more info][sysllvm].
    `sysconfdir` values are combined with the path from the `DESTDIR`
    environment variable.
 
+   If you are unsure what to use in build configurations and need a good default one,
+   you will likely find `x.py setup` more user-friendly. It's an interactive command that
+   offers a better experience compared to the `configure` script, and guides you through
+   selecting a config profile, setting up LSP, configuring a git hook, etc.
```

Comment:
What is your expectation that *most* people should use? If this is the more user-friendly method, shouldn't it be mentioned first?

Reply:
I think `x.py setup` is the one most people will be interested on, so I moved it to top.